packages/cli/tsconfig.json

```diff
@@ -1,10 +1,14 @@
 {
```

Comment:
Please revert this change so that the config extends the root config; override as necessary.